restclient/guble_sender.go

```diff
@@ -12,6 +12,11 @@ import (
 	log "github.com/Sirupsen/logrus"
 )
 
+const (
+	xHeaderPrefix        = "x-guble-"
```

Comment:
the existing const from the "rest" module could be made public, and reused here in this "restclient" package